docs/source/cuda/cuda_compilation.rst

```diff
@@ -4,30 +4,42 @@
 Compiling Python functions for use with other languages
 =======================================================
 
-Numba can compile Python code to PTX so that Python functions can be
+Numba can compile Python code to PTX or LTO-IR so that Python functions can be
 incorporated into CUDA code written in other languages (e.g. C/C++).  It is
 commonly used to support User-Defined Functions written in Python within the
 context of a library or application.
 
-The main API for compiling Python to PTX can be used without a GPU present, as
-it uses no driver functions and avoids initializing CUDA in the process. It is
-invoked through the following function:
+The compilation API can be used without a GPU present, as it uses no driver
+functions and avoids initializing CUDA in the process. It is invoked through
+the following function:
 
-.. autofunction:: numba.cuda.compile_ptx
+.. autofunction:: numba.cuda.compile
    :noindex:
 
-If a device is available and PTX for the compute capability of the current
+If a device is available and code for the compute capability of the current
```

Comment:
Is there a word other than `code` that could be used here as the output isn't "code" in the traditional sense? However, I'm not sure that there is a term that covers PTX and LTO-IR, seems like later on in the PR `compiled code` is used.

Reply:
I'm not sure I agree that it's not "code" in the traditional sense - I think "code" is a very broad term that refers to any source or executable sequence of bits.

However, I think you're asking here that I disambiguate between the input code and the output code, and I think that's helpful to do - I think changing this to "compiled code" should do that.

Reply:
Ah yes, perhaps input vs output is more of my concern opposed to the specific meaning of code in this context. Changing to "compiled code" for the output seems to be an agreeable change, thanks!